utils/validationRules/Replies/flagReplySchema.js

```diff
@@ -0,0 +1,25 @@
+const Joi = require("@hapi/joi");
+
+/**
+ * Schema validation for PATCH '/comments/{commentId}/replies/{replyId}/flag'
+ */
+const flagReplySchema = {
+  options: {
+    allowUnknown: true,
+  },
+
+  header: Joi.object({
```

Comment:
Please update this to use the plural version of header.

Reply:
Alright noted
Will do